deposit/src/main/resources/service-context.xml

```diff
@@ -175,4 +175,55 @@
 	<bean id="pooledHttpClient" class="org.apache.http.impl.client.CloseableHttpClient"
 			factory-bean="pooledHttpClientBuilder" factory-method="build">
 	</bean>
+	
+	<bean id="sparqlQueryService" class="edu.unc.lib.dl.sparql.FusekiSparqlQueryServiceImpl">
+		<property name="fusekiQueryURL" value="${fuseki.baseUri}" />
+	</bean>
+	
+	<!-- Initialize access control dependencies -->
+	<bean id="contentPathFactory" class="edu.unc.lib.dl.fedora.ContentPathFactory"
+			init-method="init">
+		<property name="cacheMaxSize" value="100" />
+		<property name="cacheTimeToLive" value="100" />
+		<property name="queryService" ref="sparqlQueryService" />
```

Comment:
Sorry I'm not sure which tag you're referring to?  Although it looks like I need to set the cache properties with injected values rather than hard coded ones